config/bootstrap.php

```diff
@@ -22,6 +22,13 @@
     class_alias('Cake\Utility\Text', 'Cake\Utility\String');
 }
 
+// @deprecated Backward compatibility with 2.x, 3.0.x
+class_alias('Cake\Mailer\AbstractTransport', 'Cake\Network\Email\AbstractTransport');
+class_alias('Cake\Mailer\Transport\DebugTransport', 'Cake\Network\Email\DebugTransport');
+class_alias('Cake\Mailer\Email', 'Cake\Network\Email\Email');
+class_alias('Cake\Mailer\Transport\MailTransport', 'Cake\Network\Email\MailTransport');
+class_alias('Cake\Mailer\Transport\SmtpTransport', 'Cake\Network\Email\SmtpTransport');
```

Comment:
In the example code it will autoload, a 3rd param of class_alias would be needed to set to false to turn off autoloading